hw/ip/otbn/util/shared/syntax.py

```diff
@@ -68,15 +68,24 @@ def asm_pattern(self) -> str:
         # We also split out ++ and -- separately, to disambiguate things like
         # x1++, which must be parsed as x1 followed by ++.
         #
+        # However, we do want to allow things like ".+123". To get this right,
+        # suppose C matches any character other than elements of " ,+-". Then
```

Comment:
```suggestion
        # suppose S matches any character other than elements of " ,+-". Then
```

?

Reply:
Ah, yes, thanks. I couldn't decide what to call it and ended up with S for **s**ymbol constituent.